Description: relax dependency on crate clap
 Needed to match Debian-packaged clap v3.2.16.
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2022-08-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -10,7 +10,7 @@
 
 [dependencies]
 anyhow = "1.0.53"
-clap = ">= 3.0.14, < 3.1.0"
+clap = "3.0.14"
 fern = { version = ">= 0.5.0, < 0.7.0", features = ["colored"] }
 globset = "0.4.8"
 ignore = "0.4.18"
